modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -15,26 +15,49 @@ class UploadsController < ApplicationController
         '10-7959A' => 'vha_10_7959a'
       }.freeze
 
-      def submit
-        Datadog::Tracing.trace('Start IVC File Submission') do
-          form_id = get_form_id
-          Datadog::Tracing.active_trace&.set_tag('form_id', form_id)
-          parsed_form_data = JSON.parse(params.to_json)
-          file_paths, metadata = get_file_paths_and_metadata(parsed_form_data)
-          statuses, error_message = FileUploader.new(form_id, metadata, file_paths, true).handle_uploads
-          response = build_json(Array(statuses), error_message)
+      if Flipper.enabled?(:champva_file_recreate, @user)
```

Comment:
This feature toggle needs to be defined in `config/features.yml`

Reply:
Looks like that will happen once [this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/19236) is merged

Reply:
Yup. It is defined in config/features.yml. Search for "champva_file_recreate"